README.md

```diff
@@ -38,6 +38,7 @@ Using ansible galaxy requirements file:
 
 | Name                           | Default               | Type   | Description              |
 | ------------------------------ | --------------------- | ------ | ------------------------ |
+| `manala_redis_version`         | ~ (auto detect)       | string | Redis installed version. |
```

Comment:
Fyi, in php role, version variable can be string, int, float or whatever. The fact is i *always* pipe it to string when using
```
manala_redis_version|string
```

I think, this could be a good practise